### PR TITLE
Fewer prints

### PR DESCRIPTION
**Pull request by pgdr**
My deepest apologies for big diffs, I've got a bad case of OCD.

Will revert and remove commits if nobody wants to review this mess.

### CODE REVIEW

File: libenkf/src/gen_data_config.c [hunk 1]
```diff
@@ -397,6 +397,49 @@ void gen_data_config_assert_size(gen_data_config_type * config , int data_size,
...
+    config->mask_modified = true;
+  }
+
+  if (!config->mask_modified)
+    return; // nothing to do
+
+  // The global mask has been modified after the last load;
+  // i.e. we update the on-disk representation.
+  char * filename = util_alloc_sprintf("%s_active", config->key);
+  FILE * stream = enkf_fs_open_case_tstep_file(
```
Comment (joakim-hove) [inline libenkf/src/gen_data_config.c]: The mother of all irrelevant comments; I would like if this was formatted with:

1. The first argument, `forward_load_context_get_...` one line up.
2. Everything indented to the column of the opening `(`

That way I think it is easier to see that this is indeed one (too) long function call, and not regular lines of code.

File: libenkf/src/gen_data_config.c [hunk 2]
```diff
@@ -407,77 +450,40 @@ void gen_data_config_assert_size(gen_data_config_type * config , int data_size,
...
-bool gen_data_config_has_active_mask( const gen_data_config_type * config , enkf_fs_type * fs , int report_step) {
-  bool   has_mask;
-  {
-    char * filename = util_alloc_sprintf("%s_active" , config->key );
-    FILE * stream   = enkf_fs_open_excase_tstep_file( fs , filename , report_step);
+bool gen_data_config_has_active_mask(const gen_data_config_type * config,
+                                     enkf_fs_type * fs,
+                                     int report_step) {
+  char * filename = util_alloc_sprintf("%s_active" , config->key );
+  FILE * stream = enkf_fs_open_excase_tstep_file(fs, filename, report_step);
```
Comment (joakim-hove) [inline libenkf/src/gen_data_config.c]: In situations where several resources are aquired and then released again I think it is a good pattern to release in opposite order - i.e. in this case that would be:
```C
char * filename = util_alloc...
FILE * stream = enkf_fs_ ..
...
...
fclose(stream);
free(filename)
```

**Comment by joakim-hove** [comment]
Some minor comments - address them if you agree.

Since it is marked as WIP I have not approved it; but modulo the test failure I would be happy to merge it in the current state. 

I go offline now - so I leave it to you; or someone else to push final green button.

**Comment by pgdr** [comment]
Rebased, but Travis seems red to me, I don't know if it's me, since I can't reproduce locally.

**Comment by pgdr** [comment]
Killed in #255

**Pull request closed without merging**